graph/telemetry/istio/appender/health.go

```diff
@@ -182,7 +183,7 @@ func (a *HealthAppender) attachHealth(trafficMap graph.TrafficMap, globalInfo *g
 
 		var req healthRequest
 		var ok bool
-		if req, ok = healthReqs[n.Namespace]; !ok {
+		if req, ok = healthReqs[n.Namespace+n.Cluster]; !ok {
```

Comment:
Minor nit, across the graph appender code, when needing a key like this, the format used has been `cluster:namespace`.  In general, it's safer to use a delimiter when combining strings, and conventionally the cluster has been left of the namespace in the graph-related keys.